BuffTheGroupMenu.lua

```diff
@@ -22,47 +20,84 @@ function BuffTheGroup.buildMenu()
 			type = "checkbox",
 			name = "Enabled",
 			tooltip = "Toggles the UI",
+			default = btg.defaults.enabled,
 			getFunc = function() 
 				return btg.savedVars.enabled
-			 end,
+			end,
 			setFunc = function(value)
 				btg.savedVars.enabled = value
 				btg.CheckActivation()
 			end,
-			
+			reference = "btgControlEnabled"
 		},
-        {
-			type = "dropdown",
-			name = "Buff",
-			tooltip = "Buff to track",
-			choices = BuffTheGroupData.buffSelectionList,
-			default = BuffTheGroupData.buffSelectionList[1],
-			getFunc = function() return BuffTheGroupData.buffSelectionList[btg.savedVars.trackedBuff] end,
-			setFunc = function(selected)
-				for index, name in ipairs(BuffTheGroupData.buffSelectionList) do
-					if name == selected then
-						btg.savedVars.trackedBuff = index
-						break
-					end
-				end
+		{
+			type = "checkbox",
+			name = "Debug Mode",
```

Comment:
Recommend renaming this to "Always On Mode" and preventing debug messages from appearing in chatbox.